firmware/controllers/map_averaging.cpp

```diff
@@ -193,6 +213,17 @@ static void mapAveragingCallback(trigger_event_e ckpEventType,
 		return;
 	}
 
+	if (boardConfiguration->mapMinBufferLength != mapMinBufferLength) {
+		// check range
+		mapMinBufferLength = maxI(minI(boardConfiguration->mapMinBufferLength, MAX_MAP_BUFFER_LENGTH), 0);
```

Comment:
how about maxI(,,,,, 1)? as long as mapMinBufferLength is at least 1, we would not need "mapMinBufferLengt > 1" checks since mapMinBufferLengt = 1 would work same exact was as mapMinBufferLengt = 2, and there would be no possibility for mapMinBufferLengt = 0?

this case we have a bit less code